mta/mta.sw.yaml

```diff
@@ -187,5 +189,28 @@ states:
     stateDataFilter:
       output: '{workflowOptions: {migrationOptions: [{id: "m2k", name: "Move2Kube workflow"}]}, repositoryURL: .repositoryURL}'
     end: true
-
+  - name: NotifyFailureBackstage
+    type: operation
+    actions:
+      - condition: .backstageUser != null
+        functionRef:
+          refName: createNotification
+          arguments:
+            title: "MTA analysis failed"
+            message: '"MTA analysis for " + .application.repository.url + " failed. Task pod: " + .taskgroup.pod +" ; Error: "+ .taskgroup.error'
+            origin: "MTA Analysis Workflow"
+            topic: "MTA Analysis Workflow"
+            targetUsers:
+              - .backstageUser
+      - condition: .backstageGroup != null
```

Comment:
I think we can shrink this to a single operation where we send both targetGroups and targetUsers arguments:
```
condition: .backstageUser != null || backstageGroup != null
...
targetUsers:
 - if .backstageUser != null  then .backstageUser else "" end
 targetGroups:
 - if .backstageGroup != null then .targetGroup else "" end 
```

I really hope the notification API can work with empty/nulls. I think it was better if the API could take bot users and group uuids. different discussion though

Reply:
I'll test to set the target to empty string or null and see how the notifications plugin react to that

Reply:
so, no it does not work, if empty or null is given, the notification plugin does not like it